src/app-layout/notifications/styles.scss

```diff
@@ -4,9 +4,11 @@
 */
 
 @use '../../internal/styles/tokens' as awsui;
+@use '../../internal/generated/custom-css-properties/index.scss' as custom-props;
```

Comment:
We generally don't use file extensions in imports, is this necessary?

Reply:
Removed